src/app/messaging/component/message-window/message-window.component.ts

```diff
@@ -55,6 +61,20 @@ export class MessageWindowComponent implements OnInit, OnDestroy {
           (messages => this.data = messages);
   }
 
+  sendMessage(){
+    let newMessage=new CreateMessage
```

Comment:
I'm not familiar with this syntax. Is this how to create an instance of CreateMessage in TypeScript?

Reply:
done. the previous syntax is for class but it now uses an interface